sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/README.md

```diff
@@ -24,6 +24,16 @@ The Azure Monitor Distro is a distribution of the .NET OpenTelemetry SDK with in
 
 * [Logs](https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/docs/logs/getting-started-console)
 
+* Resource Detectors
+  * **AppServiceResourceDetector**: Adds resource attributes for the applications running in Azure App Service.
+  * **AzureVMResourceDetector**: Adds resource attributes for the applications running in an Azure Virtual Machine.
+  * **AzureContainerAppsResourceDetector**: Adds resource attributes for the applications running in Azure Container Apps.
+
+   **Note**: The detectors are part of the [OpenTelemetry.ResourceDetectors.Azure](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.ResourceDetectors.Azure) package. While this package is currently in its beta phase, we have chosen to vendor in the code for these detectors to include them in our Distro.
```

Comment:
Should we specify the detectors are only used to set cloud role and roleinstance. All other attributes are ignored.

Reply:
Added one liner here, we should aim to cover more details in the official document.